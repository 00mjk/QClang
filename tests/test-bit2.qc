int main() {
    qubit q;
    measure(q);
    return 0;
}
