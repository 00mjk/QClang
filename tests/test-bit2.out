OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
creg c[1];
h q;
qreg n_q_q0[1];
creg n_q_q0_mb0[1];
measure n_q_q0 -> n_q_q0_mb0;
